.config/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -v -rf --ff
+testpaths =
+    tests/unit_tests*
+norecursedirs = xarray_utils
```

Comment:
Does the --cov argument go in here?  Or is that not possible because you have to include the actual package name with --cov ?

Reply:
Its possible, I just had errors yesterday, do we automatically install pytest and pytest-cov as dependencies, or should I also add them to the package list?

Reply:
If they are not there,  add them to pyproject.toml under optional test dependencies

Reply:
Already listed as optional dependencies, fixed config file to include coverage reporting.